module/src/sandbox.rs

```diff
@@ -85,6 +85,8 @@ pub trait Sandboxer: Send + Sync {
 pub trait Sandbox: Linkable {
     /// Returns the `Sandboxer` for this sandbox.
     fn sandboxer(&self) -> Arc<dyn Sandboxer>;
+
+    fn debug(&mut self, arg: &[u8]) -> Vec<u8>;
```

Comment:
It seems that `debug` is not a mandatory function for the `Sandbox`.
How about giving a default implementation?

Reply:
Done